From 3b5e69024c21356d24862da18fad1daddbe2ef8f Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 28 Nov 2023 14:22:46 +0100
Subject: [PATCH 4/5] objtool: respect AWK setting

AWK= is not passed on as an makefile argument so we need pass it as an env.

Signed-off-by: fossdd <fossdd@pwned.life>
---
 tools/objtool/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/objtool/Makefile b/tools/objtool/Makefile
index 8c20361dd..889e2e0c0 100644
--- a/tools/objtool/Makefile
+++ b/tools/objtool/Makefile
@@ -43,7 +43,7 @@ OBJTOOL_CFLAGS += $(if $(elfshdr),,-DLIBELF_USE_DEPRECATED)
 # Always want host compilation.
 HOST_OVERRIDES := CC="$(HOSTCC)" LD="$(HOSTLD)" AR="$(HOSTAR)"
 
-AWK = awk
+AWK ?= awk
 MKDIR = mkdir
 
 BUILD_ORC := n
-- 
2.49.0

